packages/vue-cli-plugin-typescript/generator/template/src/components/Pager.vue

```diff
@@ -30,8 +29,6 @@ export default Vue.extend({
     return {
       pager: pager,
       state: {...pager.state, total: 0},
-      totalCount: (this.$root.$data.$engine as SearchEngine).state.search
```

Comment:
`totalCount` was always only assigned once. Since the parameter is transmitted by value and not by reference, `totalCount` remains unchanged on future the state refreshes.